cmd/bake/bake.go

```diff
@@ -40,16 +45,13 @@ func NewBakeCommand() *cobra.Command {
 		Short: "Use a pizza-oven to source git commits into OpenSauced",
 		Long:  bakeLongDesc,
 		Args: func(cmd *cobra.Command, args []string) error {
-			if len(args) > 1 {
-				return errors.New("only a single url can be ingested at a time")
-			}
-			if len(args) == 0 {
+			if len(args) == 0 && opts.FilePath == "" {
 				return errors.New("must specify the URL of a git repository")
```

Comment:
This error string needs updating:

```suggestion
				return errors.New("must specify the URL(s) of a git repository or provide a batch file")
```